src/renderer/contexts/DependencyContext.tsx

```diff
@@ -531,29 +581,34 @@ export const DependencyProvider = memo(({ children }: React.PropsWithChildren<un
                                 </Text>
-                                <HStack>
+                                <HStack gap={2}>
                                     <HStack>
-                                        <Switch
-                                            isChecked={usePipDirectly}
-                                            isDisabled={isRunningShell}
-                                            onChange={() => {
-                                                setUsePipDirectly(!usePipDirectly);
-                                            }}
-                                        />
-                                        <Text>Use Pip Directly</Text>
-                                        <Tooltip
-                                            hasArrow
-                                            borderRadius={8}
-                                            label="Disable progress bars and use pip to directly download and install the packages. Use this setting if you are having issues installing normally."
-                                            maxW="auto"
-                                            openDelay={500}
-                                            px={2}
-                                            py={0}
-                                        >
-                                            <Center>
-                                                <Icon as={BsQuestionCircle} />
-                                            </Center>
-                                        </Tooltip>
+                                        <HStack>
+                                            <Select
+                                                isDisabled={isRunningShell}
+                                                size="md"
+                                                value={installMode}
+                                                onChange={(e) => {
+                                                    setInstallMode(e.target.value);
+                                                }}
+                                            >
+                                                <option>{installModes.NORMAL}</option>
+                                                <option>{installModes.DIRECT_PIP}</option>
+                                                <option>{installModes.MANUAL_COPY}</option>
+                                            </Select>
+                                            <Tooltip
+                                                hasArrow
+                                                borderRadius={8}
+                                                label="Change the dependency install mode. Normal Install is recommended, but if you are having issues, you can try using the Direct Pip mode (which will not show progress), or Manual/Copy mode, which will copy the correct install command to your clipboard for you to run in your own terminal (restart chaiNNer manually after)."
```

Comment:
I think this should be fancier.

![image](https://github.com/chaiNNer-org/chaiNNer/assets/20878432/ef5e9f17-91cd-43b0-8401-3b781a2e1c33)
```suggestion
                                                label={
                                                    <Markdown nonInteractive>
                                                        {'The dependency install mode. ChaiNNer supports 3 ways of installing packages:\n\n' +
                                                            '- Normal: This is the default installation mode. If you are having issues, you can try using the Direct Pip.\n' +
                                                            '- Direct Pip: This will invoke pip more directly, which can fix some issues, but also makes it impossible to show installation progress.\n' +
                                                            '- Manual/Copy: Copy the pip install command to your clipboard for you to run in your own terminal. You have to manually restart chaiNner afterwards.'}
                                                    </Markdown>
                                                }
```